core/src/nucleus/ribosome/api/commit.rs

```diff
@@ -2,32 +2,40 @@ use crate::{
     nucleus::ribosome::{api::ZomeApiResult, Runtime},
     workflows::author_entry::author_entry,
 };
-use holochain_core_types::{cas::content::Address, entry::Entry, error::HolochainError};
+use holochain_core_types::error::HolochainError;
+
+use holochain_wasm_utils::api_serialization::commit_entry::{CommitEntryArgs, CommitEntryResult};
+
 use std::convert::TryFrom;
 use wasmi::{RuntimeArgs, RuntimeValue};
 
-/// ZomeApiFunction::CommitAppEntry function code
+/// ZomeApiFunction::CommitArgs function code
```

Comment:
This seems like a mistaken change to the comment.

Reply:
Reverted back in 87733210074c12bca9177c4ad22e81a8cc35dfe7.